tests/events/testUpdateEvent.py

```diff
@@ -96,3 +97,17 @@ def test_updateDate(self):
 
         retrieved_event = self.client.events.retrieve(event.key)
         assert_that(retrieved_event.date).is_equal_to(date(2023, 1, 10))
+
+    def test_updateIsInThePast(self):
+        chart = self.client.charts.create()
+        self.client.seasons.create(chart.key, event_keys=["event1"])
+
+        self.client.events.update("event1", is_in_the_past=True)
+        retrieved_event = self.client.events.retrieve("event1")
+        assert_that(retrieved_event.is_in_the_past).is_true()
+
+        try:
```

Comment:
I wouldn't test events in the past being immutable - that's covered by the tests in `seatsio-core-v2`.